src/options/components/ghostery-options.js

```diff
@@ -14,32 +14,19 @@ import { t } from '/vendor/@whotracksme/ui/src/i18n.js';
 
 const Options = {
   trackerWheelDisabled: false,
+  wtmSerpReport: true,
   [store.connect]: {
     get: async () => {
       const storage = await chrome.storage.local.get(['options']);
-      const options = storage.options || {};
-      if (typeof options.trackerWheelDisabled === 'undefined') {
```

Comment:
The model is automatically initialized with default values. It is not required to pass values explicitly.